src/winetricks

```diff
@@ -6632,13 +6632,18 @@ load_dxsdk_jun2010()
 #----------------------------------------------------------------
 
 # $1 - dxvk archive name (required)
-# $2 - minimum Wine version (optional)
-# $3 - minimum Vulkan API version (optional, requires $2 be set)
+# $2 - dxvk d3d10 support 0=no / 1=yes (required)
```

Comment:
Since this value is only used by the helper (and not, e.g., set in the registry), I'd prefer something more descriptive. E.g., d3d10_enabled / d3d10_disabled (doesn't have to be exactly that, if you've got better ideas).